spec/helper/page.js

```diff
@@ -27,7 +25,7 @@ async function initLoader(page, channelId = undefined, withBanner = false) {
         : ""
     }`,
   );
-  await wait(500);
+  const isLoaded = page.waitForFunction(() => document.readyState === "complete");
   return isLoaded;
 }
 
```

Comment:
I think it would be good to extend the related test as well to have two subscribers and verify that each are called on each events.

Reply:
I've added a test for the case of registering an additional onLogEvent callback